helpers/middlewares.js

```diff
@@ -0,0 +1,8 @@
+exports.exceptionMiddleware = async function exceptionMiddleware(err, req, res, next) {
+    res.status(500);
+    res.send({
+        'contextId': err.contextId,
+        'pageId': err.pageId,
+    });
```

Comment:
What do you think of sending error messages, to show them in crawler's logs?

Reply:
Well, I don't think that crawlers need error messages as they can see this in service's logs (can they?).
One more reason NOT to add this feature is that we would print log-messages with other log-messages such as RETRY. So, we would get too many logs.
Also I'm not sure if we want to process all bad responses on our own.
At least for now I add 'error_msg' to response body in Service. You may delete it if you want.

Reply:
> I don't think that crawlers need error messages as they can see this in service's logs (can they?).

In general crawlers do not have access to service logs.

> One more reason NOT to add this feature is that we would print log-messages with other log-messages such as RETRY. So, we would get too many logs.

I believe it is ok to log them with debug level, as retry middleware does

> Also I'm not sure if we want to process all bad responses on our own.

I agree that not all errors are meaningful to crawlers and/or should be shown to users. I'd suggest to define appropriate errors for each request/action. But for now I'm ok with the current solution.